Description: fix compilation error on big endian platforms
 /usr/bin/ld: perfunwind.o: in function `QDataStream::Version qToLittleEndian<QDataStream::Version>(QDataStream::Version)':
 undefined reference to `QDataStream::Version qbswap<QDataStream::Version>(QDataStream::Version)'
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: not-yet
Last-Update: 2020-06-25

--- a/src/tools/perfparser/app/perfunwind.cpp
+++ b/src/tools/perfparser/app/perfunwind.cpp
@@ -127,7 +127,7 @@ PerfUnwind::PerfUnwind(QIODevice *output
         // Write minimal header, consisting of magic and data stream version we're going to use.
         const char magic[] = "QPERFSTREAM";
         output->write(magic, sizeof(magic));
-        qint32 dataStreamVersion = qToLittleEndian(QDataStream::Qt_DefaultCompiledVersion);
+        qint32 dataStreamVersion = qToLittleEndian(static_cast<qint32>(QDataStream::Qt_DefaultCompiledVersion));
         output->write(reinterpret_cast<const char *>(&dataStreamVersion), sizeof(qint32));
     }
 }
